lib/middlewares.ts

```diff
@@ -46,7 +47,7 @@ export const webhookMiddleware = (
 export const jwtVerifyMiddleware = async (request: NextApiRequest) => {
```

Comment:
I think you can replace this middleware with `app-sdk` as well: there's [`withWebhookSignatureVerified` middleware](https://github.com/saleor/saleor-app-sdk/blob/main/src/middleware.ts#L62)

Reply:
I've already used it in `saleor-app-checkout`, you can take a look: https://github.com/saleor/saleor-checkout/pull/145/files#diff-49e1d61d1e40aa589a496de1ad4bec8cc277aae584ccc1ec0fc7f37029d31909 😉

Reply:
@witoszekdev Yeah, I saw that. I was thinking about porting this in separate PR, when I will be working on adding support for tax sync webhooks, as it will be mandatory there but I can add it here if you want to

Reply:
@witoszekdev I just checked it once again. I can't use it in that way as you showed. I am veryfing a token that I was recieved from iframe.  
The `withWebhookSignature...` is dedicated to validating a payload of webhooks, which use a payload, signature and public key to make sure that payload comes from correct Saleor.

Reply:
OK, that makes sense. But if possible `app-sdk` should also have a middleware to make this sort of verification - you could add an issue in that repository and describe your use case ;)

Reply:
Right, I've created an issue for this - https://github.com/saleor/saleor-app-sdk/issues/10